Makefile

```diff
@@ -91,7 +91,7 @@ lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
 
 .PHONY: yamllint
 yamllint:
-	@docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest testdata -d "{extends: relaxed, rules: {line-length: {max: 120}}}" --no-warnings
+	@docker run --rm $$(tty -s && echo "-it" || echo) -v $(PWD):/data cytopia/yamllint:latest testdata -d "{extends: relaxed, rules: {line-length: {max: 120}}, ignore: /testdata/project-v2/}"
```

Comment:
kustomize/v1 we should ignore as well
It is deprecated. So that we should no longer change
Could you please revert the changes made into common/kustomize/v1 and rerun make generate to update the samples to push another PR?

Reply:
Sure. I just applied your suggestions.
Does it work for you now?